README.md

```diff
@@ -1 +1,86 @@
-# hello_rails_back_end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello, world! Back-end
+
+> This API-only app displays a random greeting message. It is considered the back-end.
+
+## Built With
+
+- Ruby & PostgreSQL
+- Ruby on Rails
+- Linters (Rubocop & Stylelint)
+- Git & GitHub
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+    Ruby 3.1.2
+    PostgreSQL 15.0
+    Ruby on Rails 7.0.4
+
+### Setup
+
+    git clone git@github.com:jevazquezb/hello_rails_back_end.git
+    cd hello_rails_back_end
+    
+
+### Install
+
+    1. bundle install
+    2. rails db:create
+    3. rails db:migrate
+
+### Feed
+
+**Note:** Repeat step 5 to add several messages to the API (change the message).
+
+    4. rails c
+    5. Greeting.create(message: 'Hello, world!')
```

Comment:
- [ ] OPTIONAL
Both frontend and backend connect perfectly fine, however, your greetings need to be generated **randomly** when the page is refreshed or so. To achieve the same, I therefore suggest you add other random greeting messages in addition to _step 5, under Feed, of your ReadMe.